tests/e2e/framework/kubernetes.go

```diff
@@ -138,6 +138,19 @@ func (k *KubeInfo) Setup() error {
 func (k *KubeInfo) Teardown() error {
 	glog.Info("Cleaning up kubeInfo")
 	var err error
+
+	if *rbacfile != "" {
+		baseRbacYaml := util.GetResourcePath(*rbacfile)
+		testRbacYaml := filepath.Join(k.TmpDir, "yaml", filepath.Base(*rbacfile))
+		if err := k.generateRbac(baseRbacYaml, testRbacYaml); err != nil {
```

Comment:
You don't need to generate it again. You just need to check if `testRbacYaml` is there. If yes, kubeDelete it.

Reply:
done, PTAL